src/software/world/field.h

```diff
@@ -355,6 +355,42 @@ class Field
      */
     bool pointInEnemyDefenseArea(const Point &p) const;
 
+    /**
+     * Returns true if the ball is in the friendly half of the field, and false otherwise
+     *
+     * @param ball The ball
+     * @return true if the ball is in the friendly half of the field, and false otherwise
+     */
+    bool pointInFriendlyHalf(const Point &p) const;
+
+    /**
+     * Returns true if the ball is in the enemy half of the field, and false otherwise
+     *
+     * @param ball The ball
+     * @return true if the ball is in the enemy's half of the field, and false otherwise
+     */
+    bool pointInEnemyHalf(const Point &p) const;
+
+    /**
+     * Returns true if the ball is in within the provided radius in one of the friendly
+     * corner.
+     *
+     * @param ball The ball
+     * @param the radius from the corner, to decide whether or not the ball is in the
+     * field.
+     */
+    bool pointInFriendlyCorner(const Point &p, double radius) const;
+
+    /**
+     * Returns true if the ball is in within the provided radius in one of the enemy
+     * corner.
+     *
+     * @param ball The ball
```

Comment:
can you change this to point instead of ball, there is no ball here

Reply:
Forgot to update the javadoc, thanks for the heads up!